src/build_targets.rs

```diff
@@ -131,7 +131,9 @@ impl BuildTargets {
     pub fn shared_output_file_name(&self) -> Option<OsString> {
         match self.lib_type() {
             LibType::Windows => {
-                if self.shared_lib.is_some()
+                if self.target.os == "cygwin" {
+                    Some(format!("cyg{}.dll", self.name).into())
```

Comment:
is this really correct?

Reply:
Here as an example: https://cygwin.com/packages/x86_64/libglib2.0_0/libglib2.0_0-2.86.0-1